distribution/lib/Standard/Table/0.0.0-dev/docs/api/Spatial_Table.md

```diff
@@ -15,6 +15,9 @@
     - Touches left:(Standard.Base.Data.Text.Text|Standard.Base.Data.Numbers.Integer)= right:(Standard.Base.Data.Text.Text|Standard.Base.Data.Numbers.Integer)=
     - Within left:(Standard.Base.Data.Text.Text|Standard.Base.Data.Numbers.Integer)= right:(Standard.Base.Data.Text.Text|Standard.Base.Data.Numbers.Integer)=
 - type Spatial_Table
+    - as_spatial_column self column:Standard.Table.Column.Column -> Standard.Base.Any.Any
```

Comment:
I see no constructor for `Spatial_Table` in the API. That's what I would expect for all _refined types_.